Tutorials/bash/samtools_count_para_chr_xargs_sh/Readme.md

```diff
@@ -1,7 +1,7 @@
-This applet slices a BAM file by canonical chromosome then performs a Parallel `samtools view -c` using xargs. Type `man xargs` for general usage information.
+This applet slices a BAM file by canonical chromosome then performs a parallelized `samtools view -c` using xargs. Type `man xargs` for general usage information.
```

Comment:
"Parallel" is something that is, well, parallel. "Parallelized" is something that *has been made parallel*. See "straight" vs. "straightened".